bin/tasks/report.js

```diff
@@ -16,8 +16,9 @@ const ReportQuery = `
         cachedUrl
         createdAt
         completedAt
-        snapshots {
+        tests {
           status
+          result
```

Comment:
Did you add `result` to `testFields` in `authorizedBuildFieldsViaAppCode`?

Reply:
Good point https://github.com/chromaui/chromatic/pull/4504